app/src/main/java/com/edulectronics/tinycircuit/Models/Line.java

```diff
@@ -13,4 +15,18 @@ public Line(Point a, Point b) {
         this.a = a;
         this.b = b;
     }
+
+    public boolean isTouched(Point point){
+        float width = WireView.paint.getStrokeWidth() * 4;
+        if(point.x > a.x - width && point.y > a.y - width &&
```

Comment:
Ipv deze if/else gewoon de kleinste x en y en grootste x en y pakken, + en - de strokewith, en kijken of binnn de resulterende rechhoek is geklikt.